pkg/server/server.go

```diff
@@ -96,19 +96,30 @@ func renderer(templatePath string) multitemplate.Renderer {
 	return r
 }
 
-func requireAuth(db *sqlx.DB, secret string) func(*gin.Context) {
+func jobAuth(ctx *gin.Context, db *sqlx.DB, secret string) (string, error) {
```

Comment:
It took me a moment to realize what this function was pulling out specifically, thoughts on renaming it to something like `getJobAuthSignature`?

Reply:
How about if I change that function name and then refactor `SignatureForJob` to be a method on `Job` and rename it to `AuthSignature`? That way the functions aren't named too much alike.

Reply:
I dig it!